modules/core/src/lib/composite-layer.js

```diff
@@ -205,6 +205,12 @@ export default class CompositeLayer extends Layer {
     return newProps;
   }
 
+  _onViewportChange() {
+    if (this.needsUpdate()) {
```

Comment:
This seems like a no-op: if needs update set needs update?

Ot is it about transferring a child update flag to the local state?

Worth a comment